modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js/components/RatingsSelectStars.js

```diff
@@ -51,9 +58,13 @@ export default function RatingsSelectStars({
 						className: 'ratings-stars-dropdown',
 					}}
 					onActiveChange={(isActive) => setIsDropdownOpen(isActive)}
+					onFocus={handleInitialFocus}
+					role="listbox"
 					trigger={
 						<ClayButton
-							aria-pressed={!!score}
+							aria-expanded={isDropdownOpen}
+							aria-haspopup="listbox"
+							aria-label={getTitle()}
```

Comment:
Do we really need `aria-label={getTitle()}`? It's a duplication of the title attribute 🤔

Reply:
Based on [W3C recommendations](https://www.w3.org/WAI/tutorials/forms/labels/) if we have a look at below section:

> ### Using the title attribute
> The title attribute can also be used to identify form controls. This approach is generally less reliable and not recommended because some screen readers and assistive technologies do not interpret the title attribute as a replacement for the label element, possibly because the title attribute is often used to provide non-essential information. The information of the title attribute is shown to visual users as a tool tip when hovering over the form field with the mouse.

So if we want to be reliable for screen readers we should use `aria-label`. And if we want to show a tooltip, we need to use `title`. And if we want both, we need to use `aria-label `and `title`.

Reply:
@antonio-ortega You are right. Thanks for the info!
I'm not sure if ios better use the `data-title` instead of `title` on this case to avoid repetition but up to you.